packages/browser/__tests__/Session.spec.ts

```diff
@@ -1020,5 +1020,74 @@ describe("Session", () => {
         inIframe: true,
       });
     });
+
+    it("sets the updated session info after silently refreshing", async () => {
+      const sessionId = "mySession";
+      mockLocalStorage({
+        [KEY_CURRENT_SESSION]: sessionId,
+      });
+      mockLocation("https://mock.current/location");
+      const mockedStorage = new StorageUtility(
+        mockStorage({
+          [`${USER_SESSION_PREFIX}:${sessionId}`]: {
+            isLoggedIn: "true",
+          },
+        }),
+        mockStorage({})
+      );
+      const clientAuthentication = mockClientAuthentication({
+        sessionInfoManager: mockSessionInfoManager(mockedStorage),
+      });
+      const validateCurrentSessionPromise = Promise.resolve({
+        issuer: "https://some.issuer",
+        clientAppId: "some client ID",
+        clientAppSecret: "some client secret",
+        redirectUrl: "https://some.redirect/url",
+        tokenType: "DPoP",
+      });
+      clientAuthentication.validateCurrentSession = jest
+        .fn()
+        .mockReturnValue(validateCurrentSessionPromise);
+      const incomingRedirectPromise = Promise.resolve({
+        isLoggedIn: true,
+        webId: "https://some.pod/profile#me",
+        sessionId: "someSessionId",
```

Comment:
```suggestion
        sessionId: "someSessionId",
        expirationDate: 961106400
```
This sets a new expiration date for the access token held by the session. It happens to be the release date of Pokemon yellow in Europe, arguably an excellent day, but that's purely coincidental.

Reply:
4134dc9f